client/me/purchases/remove-purchase/remove-domain-dialog/index.jsx

```diff
@@ -214,36 +179,41 @@ class RemoveDomainDialog extends Component {
 			return;
 		}
 		this.props.closeDialog();
-		this.setState( { step: 1 } );
+		this.setState( { domainValidated: false, step: 1 } );
 	};
 
 	render() {
 		const { purchase, translate, chatButton } = this.props;
 		const productName = getName( purchase );
+
+		// To be removed once "Never mind" is translated; the typo has been fixed in existing translations.
+		const oldString = translate( 'Nevermind' );
+		const newString = translate( 'Never mind' );
+		const closeDialogString = hasTranslation( 'Never mind' ) ? newString : oldString;
```

Comment:
Can you switch this to just `const closeDialogString = translate( 'Never mind' );` now that we have to do the string freeze thing anyway (see my overall review comment)?

For future reference, by the way, it looks like the code here results in "Nevermind" being shown in English (apparently you need to add a `locale.startsWith( 'en' )` check in there to avoid that).  Also, as a side note, we have a number of languages that rarely get translated in practice and mostly fall back to English, so with this approach we would need to eventually go back in and remove this `hasTranslation()` code later on (so that those customers eventually see the updated English wording also)... Interestingly, I guess your original approach here would have avoided both those issues (although only works because we "know" that the "Nevermind" typo wouldn't have propagated to other languages)!

Reply:
Thanks @DavidRothstein - I've switched it up as you suggested! Yep, that makes sense - I was relying on the `Nevermind` string to be correctly translated previously, which is why I also thought that'd work fine. But this approach is definitely better for ensuring the new string is also translated and sparing another PR! :)